From f9a84474cae6a4e2e1d08a739dd867f2105c6c74 Mon Sep 17 00:00:00 2001
From: Richard Guy Briggs <rgb@redhat.com>
Date: Thu, 10 Apr 2014 14:57:45 -0400
Subject: [kernel] audit: fix AUDIT_FEATURE_CHANGE record number

Message-id: <2820acb40ac2004c1f902bdf6900db3f69ee6254.1397141675.git.rgb@redhat.com>
Patchwork-id: 78857
O-Subject: [RHEL7.1 PATCH] audit: fix AUDIT_FEATURE_CHANGE record number
Bugzilla: 1085928
RH-Acked-by: Eric Paris <eparis@redhat.com>
RH-Acked-by: Paul Moore <pmoore@redhat.com>
RH-Acked-by: Steve Grubb <sgrubb@redhat.com>

The 1000 block is for audit commands.

Use the 1300 block for logging events.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1085928
BREW: https://brewweb.devel.redhat.com/taskinfo?taskID=7323303

Signed-off-by: Richard Guy Briggs <rgb@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/audit.h b/include/uapi/linux/audit.h
index 9539ea9..ec7a34b 100644
--- a/include/uapi/linux/audit.h
+++ b/include/uapi/linux/audit.h
@@ -70,7 +70,6 @@
 #define AUDIT_TTY_SET  1017 /* Set TTY auditing status */
 #define AUDIT_SET_FEATURE 1018 /* Turn an audit feature on or off */
 #define AUDIT_GET_FEATURE 1019 /* Get which features are enabled */
-#define AUDIT_FEATURE_CHANGE 1020 /* audit log listing feature changes */
 
 #define AUDIT_FIRST_USER_MSG 1100 /* Userspace messages mostly uninteresting to kernel */
 #define AUDIT_USER_AVC  1107 /* We filter this differently */
@@ -109,6 +108,7 @@
 #define AUDIT_NETFILTER_PKT 1324 /* Packets traversing netfilter chains */
 #define AUDIT_NETFILTER_CFG 1325 /* Netfilter chain modifications */
 #define AUDIT_SECCOMP  1326 /* Secure Computing event */
+#define AUDIT_FEATURE_CHANGE 1328 /* audit log listing feature changes */
 
 #define AUDIT_AVC  1400 /* SE Linux avc denial or grant */
 #define AUDIT_SELINUX_ERR 1401 /* Internal SE Linux Errors */
-- 
1.7.1